apps/web/util/helpers/s3-helpers.ts

```diff
@@ -29,10 +29,13 @@ export const createNewBucket = async (
 };
 
 export const beforeCreatingDoc = async (req: NextApiRequest, res: NextApiResponse, body: any) => {
+
   const { data, type } = body;
 
   switch (type) {
     case "firebase":
```

Comment:
@nghiapham1026 @9brian we might want to store these as global constants in the future so it can be used across different files, i.e. in one file it might be misspelled but here it is spelled correctly

Reply:
forgot to add, but constants help prevent these inconsistencies